### ISSUE TITLE
Project display settings do not change after resetting the project settings

### ISSUE THREAD

**Issue by dbemke** [getodk/collect#5356]
#### ODK Collect version
v2021.3.4, the store version v2022.4 Beta 2, master 6e836499110deb34377a8558cbc815fdc08ca25c
#### Android version
8.1, 10, 11, 13 (probably all)
#### Device used
Nexus 5X, Redmi 9T, Galaxy M12,  Pixel 6a
#### Problem description
After resetting the project settings the changed project name, icon, color (project display in settings) don’t change values.
#### Steps to reproduce the problem
1. Go to settings -> project display.
2. Change name, icon and color of the project.
3. Go to settings -> project management -> reset.
4. Select "All settings".
5. Tap "Reset" and "ok".

#### Expected behavior
Project display settings should change.
#### Other information 
The issue is also present in v2022.3.0 and v2022.3.6.

**Comment by seadowg** [comment]
> After resetting the project settings the changed project name, icon, color (project display in settings) don’t change values.

By "changed", do you mean that the name/icon/color have been changed since creating the project from a QR code (or manually)?

**Comment by dbemke** [comment]
Both changing manually in settings and or reconfiguring via QR code after creating a project. I discussed this with Grzegorz and other settings go back to default settings (after resetting) only project display settings stays as they were before resetting. So probably the question is how project display settings should change/behave after resetting.

**Comment by seadowg** [comment]
> So probably the question is how project display settings should change/behave after resetting.

Yeah, this an interesting question. We present "Project display" settings along with all the other settings that get "reset", but then don't reset them if "All settings" is checked. There are some other thoughts that come to mind here:

1. Reset for settings is actually pretty different from the reset for forms, form cache and map layers - it restores every setting to the default value rather than deleting anything
3. What would the "default" value for each project display setting even be? Would we just regenerate them based on the server settings?
4. Should we even have "Reset" functionality now that a project can simply be deleted? There are other ways to delete forms, instances and map layers and settings resets seem far less valuable now we have projects?
5. ...unless resetting settings should actually restore the "original" settings for a project (the ones imported from the QR code or entered manually when creating the project) which would be a faster way of hitting "Reconfigure with QR code" and then scanning the original QR code again

I think we could some analytics to help us with some of these discussions:

1. Event when everything is reset
2. Events when subset of items are reset (one event for each checkbox)

I feel like if we see that people are mostly resetting everything, then we could consider removing "Reset" and making people use "Delete" instead. I think this would be worth getting more info on before we consider creating reset behaviour for project display settings.

**Comment by lognaturel** [comment]
> Should we even have "Reset" functionality now that a project can simply be deleted? There are other ways to delete forms, instances and map layers and settings resets seem far less valuable now we have projects?

Agreed deleting all instances is easy enough from the Delete menu.

I think it's still useful for forms, though. For forms, I think the most useful thing it does is (I believe) clear last-saved. I use it to clear the form cache when things are in a weird state though that may not be broadly used.

For settings, not everyone is configuring from a QR code yet. If they're not, I think clearing settings is still valuable. For example, field staff may be in a weird state. A project manager might come in and clear settings, configure as needed, and add an admin password. I think people are also not very comfortable with switching between projects so they may use the same project and reset once their tasks change.

**Comment by seadowg** [comment]
@lognaturel all good points. You've convinced me it's too early on the projects "journey" to rework this just yet. Given that, is there a good option for what resetting project display settings should actually do, or should we just think of this as "works as intended"? I'm leaning towards the latter as I'm not sure if there's an option that makes the situation less confusing (resetting back to the "original" values or the default ones both feel strange to me).

**Comment by grzesiek2010** [comment]
> Should we even have "Reset" functionality now that a project can simply be deleted?

Not always it is that simple. You have to:
- remove all unsent forms if you have any
- wait for background jobs to finish

but some users might want to quickly get rid of everything and then resetting seem to make sense.
Resetting all settings converts the project to the Demo one. In Demo project project details are Demo project/D/blue color so I think we should update it like that.
